tardis/io/logger/tests/test_logging.py

```diff
@@ -7,17 +7,34 @@
 from tardis import run_tardis
 
 
-def test_logging_simulation(atomic_data_fname, caplog):
+@pytest.fixture(scope="module")
+def config_verysimple(config_verysimple, atomic_data_fname):
     """
-    Testing the logs for simulations runs
+    Further simplify the `config_verysimple` fixture for testing logging.
     """
+    config_verysimple.montecarlo.iterations = 2
+    config_verysimple.montecarlo.no_of_packets = 400
+    config_verysimple.montecarlo.last_no_of_packets = -1
+    config_verysimple.spectrum.num = 2000
+    config_verysimple.atom_data = atomic_data_fname
+    return config_verysimple
+
+
+@pytest.fixture(scope="module")
+def config_verysimple_logger(atomic_data_fname):
     config = Configuration.from_yaml(
-        "tardis/io/tests/data/tardis_configv1_verysimple.yml"
+        "tardis/io/tests/data/tardis_configv1_verysimple_logger.yml"
     )
     config["atom_data"] = atomic_data_fname
+    return config
 
-    simulation = Simulation.from_config(config)
 
+def test_logging_simulation(caplog, config_verysimple, atomic_data_fname):
+    """
+    Testing the logs for simulations runs
+    """
+    config_verysimple["atom_data"] = atomic_data_fname
+    simulation = Simulation.from_config(config_verysimple)
     simulation.run()
```

Comment:
Just to make sure this is done, this code can be replaced by the `simulation_verysimple` fixture in the main `conftest.py` file.